src/js/polyfills/worker.js

```diff
@@ -2,14 +2,34 @@ const core = globalThis[Symbol.for('tjs.internal.core')];
 const _Worker = core.Worker;
 
 import { defineEventAttribute } from './event-target';
+import { kBlobGetParts } from './blob';
 
 const kWorker = Symbol('kWorker');
 
+function blobTextSync(blob) {
+    const decoder = new TextDecoder();
+    let str = '';
+
+    for (const part of blob[kBlobGetParts]) {
+        if (part instanceof Blob) {
+            str += blobTextSync(part);
+        } else {
+            str += decoder.decode(part, { stream: true });
+        }
+    }
+
+    str += decoder.decode();
```

Comment:
IIRC it's better to collect all parts in an array and then join it with ''

Reply:
I've just copied the implementation used in `Blob.text`